File: 67_sap_xep_chu_so.py
def solve():
    global n ,M , check
    for i in range(0 , len(M)):
        while(M[i] > 0):
            check[M[i] % 10 ]+=1
            M[i]/=10 
    for i in range(1  , 10):
        if check[i]>0 :
            print(i  , end= " ")
    print() 
t =int(input())
while t>0 :
    n = int(input()) 
    M = list(map(int , input().split())) 
    check = [0,0,0,0,0,0,0,0,0,0]
    solve()
#
    t-=1

File: 67_sap_xep_chu_so.cpp
#include<iostream>
using namespace std ; 
void solve() {
    int n  ; cin>> n ;
    int M[n] ; 
    for(int i=0 ; i<n   ; i++) cin>>M[i] ; 
    int check[10] = {0} ; 
    for(int i=0 ; i<n ; i++) {
        while(M[i]>0) {
            int k = M[i]%10 ; 
            check[k] +=1 ; 
            M[i] /=10 ; 
        }
    }
    for(int i=0 ; i<10 ; i++) {
        if(check[i] > 0 ) cout<<i<<" " ; 
    }
    cout<<endl ; 
}


main() {
    int t; cin>>t ; 
    while(t--){
        solve() ; 
    }
}